pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -201,66 +251,119 @@ func (cfg Config) toAuthOptions() gophercloud.AuthOptions {
 
 func (cfg Config) toAuth3Options() tokens3.AuthOptions {
```

Comment:
We might not need this function now that we're loading configs using `clientconfig`